tests/installation/reboot_after_installation.pm

```diff
@@ -20,7 +20,14 @@ sub run {
     # console but we never switched away, see
     # logs_from_installation_system.pm, so we should be safe to ignore this
     # call
-    select_console 'installation' unless get_var('REMOTE_CONTROLLER');
+    if (check_var('BACKEND', 'spvm')) {
+        # this will only work for serial install
+        select_console 'novalink-ssh', await_console => 0;
+        assert_screen 'rebootnow';
+    }
+    else {
+        select_console 'installation' unless get_var('REMOTE_CONTROLLER');
```

Comment:
Hmm, the "bad" thing here is that we don't use the more "standard" `'installation'` console like others backend do. That can lead to other issues in more complex tests.

I'm partially able to use `'installation'` console on my local dev, but I have issue with the `first_boot'. i will have a look with all your new changes :-)

Reply:
I don't see how I could do that.
The only way that I can think of is running the installation via SSH.
But currently I'm running it over the serial console via novalink.

Reply:
Yes, this is how `bootloader_skvm` seems to work. And it works, I "just" have issue with reboot, as it's needed to open `'novalink-ssh'` console for that part.

But all your devs are good anyway! And this can be changed after if it's possible/needed, as your current code is running well.

Reply:
@ldevulder there is no `bootloader_skvm` - did you mean `bootloader_zkvm`?

Reply:
Yes, `zkvm` :-)